Cargo.toml

```diff
@@ -10,9 +10,35 @@ readme = "README.md"
 license = "MIT/Apache-2.0"
 keywords = ["succinct", "rank", "select"]
 
+[features]
+default = []
+fuzz = ["afl"]
+simd = ["packed_simd"]
+
 [dependencies]
 num-traits = "0.2"
 byteorder = "1.2"
 
+[dependencies.afl]
+version = "0.5.2"
+optional = true
+
+[dependencies.packed_simd]
+version = "0.3"
+features = ["into_bits"]
+optional = true
+
 [dev-dependencies]
 quickcheck = "0.9.0"
+quickcheck_macros = "0.8.0"
+criterion = "0.3"
+rand = "0.7"
+
+[[bench]]
+name = "rank_select"
+harness = false
+
+[[bin]]
+name = "rsdict_fuzz"
+path = "src/rsdict/fuzz.rs"
```

Comment:
This seems like a test, not a \[\[bin\]\].

Reply:
ah, yeah I made it a binary since it runs forever, and we wouldn't want that running with `cargo test`.  I could make it run for some bounded amount of time and change it to a test or leave it as is.  what do you think?

Reply:
If I recall correctly, there’s a way to mark a test so that it isn’t run be default. If that doesn‘t work, maybe it could be an example?

(BTW, you’ve done really impressive work here. I should have said that first!)